File: test-suite/src/test/java/io/micronaut/xml/jackson/docs/Book.java
package io.micronaut.xml.jackson.docs;

import tools.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
import io.micronaut.core.annotation.Introspected;

@Introspected
@JacksonXmlRootElement(localName = "book")
public record Book(String name) {
}

File: test-suite-kotlin/src/test/kotlin/io/micronaut/xml/jackson/docs/Book.kt
package io.micronaut.xml.jackson.docs

import io.micronaut.core.annotation.Introspected
import tools.jackson.dataformat.xml.annotation.JacksonXmlRootElement

@Introspected
@JacksonXmlRootElement(localName = "book")
data class Book(val name: String)
